packages/template-retail-react-app/app/mocks/mock-data.js

```diff
@@ -2543,58 +2543,369 @@ export const mockProductSearch = {
     limit: 25,
     hits: [
         {
-            currency: 'USD',
+            currency: 'GBP',
             hitType: 'master',
             image: {
-                alt: 'Slim Skirt With Back Kick Pleats, , large',
+                alt: 'Cuffed Cargo Pant, , large',
                 disBaseLink:
-                    'https://edge.disstg.commercecloud.salesforce.com/dw/image/v2/ZZRF_001/on/demandware.static/-/Sites-apparel-m-catalog/default/dw1c5f9222/images/large/PG.10221626.JJ3WCXX.PZ.jpg',
-                link: 'https://zzrf-001.dx.commercecloud.salesforce.com/on/demandware.static/-/Sites-apparel-m-catalog/default/dw1c5f9222/images/large/PG.10221626.JJ3WCXX.PZ.jpg',
-                title: 'Slim Skirt With Back Kick Pleats, '
+                    'https://edge.disstg.commercecloud.salesforce.com/dw/image/v2/ZZRF_001/on/demandware.static/-/Sites-apparel-m-catalog/default/dwbe65be7a/images/large/PG.10226893.JJI15XX.PZ.jpg',
```

Comment:
What caused use to make such a large change this data?Seems like we are adding more than just a "orderable" value